docs/learn/learn-async-backing.md

```diff
@@ -7,103 +7,232 @@ keywords: [parachains, backing, parablock, perspective parachains, unincluded se
+filled and pushed to the unincluded segment, ready to be backed. The unincluded segment is now 2/3
+filled.
+
+Collators were able to make better use of blockspace, including 2x more data, by doubling block
+generation time from 6 to 12 seconds in a period of lower demand. This was possible due to the
+asynchrony between block generation and backing. With an unincluded segment carrying a maximum of
+three parablocks, the collators could wait an additional 6 seconds if needed (i.e. make sure in the
+unincluded segment there is at least one parablock to build a new one).
+
+In case blockspace demand increases after the generation of P7, collators can generate P8 and P9 is
+6 seconds (i.e. 3 seconds per block) due to the unincluded segment being 2/3 filled (see below).
+However, if demand stays high after P9, collators will only be able to generate parablocks every 6
+seconds as the unincluded segment is emptied every 6 seconds and its maximum capacity of 3/3
+parablocks has been reached.
+
+![async-backing-3s](../assets/async-backing-3s.png)
+
+In general, asynchronous backing will allow [collators](./learn-parachains-protocol.md#collators) to
+include an estimated ~3-5x more data into parablocks while speeding up parachain block times from 12
+to 6 seconds. Due to the 2x decrease in block time to inclusion and the possibility of building
+blocks in advance to fit more data, Polkadot with asynchronous backing can deliver an estimated
+~6-10x more blockspace to its parachains.
 
-## Synchronous Backing on Polkadot
+This combination of lower latency, higher storage per block, and a logical pipeline spanning
+Polkadot's networking, runtime, and collation aspects will allow for higher, more robust throughput.
 
-With synchronous backing, there was only about a single relay chain block (6-second window), to
-complete the parablock candidate generation and backing process. This was tightly coupled to the
-relay chain's progress, where blocks had to be created within this window.
+### Prospective Parachains
 
-The main limitation of synchronous backing is that parablock validation is tightly coupled to the
-relay chain's progression on a 1-1 basis, meaning every parablock must be generated and backed
-within six seconds. This time limit reduces the amount of data a collator has time to add to each
-block.
+Prospective parachains manage and represent all unincluded segments from all parachains.
 
-A particular parablock, `P1`, would **only** be valid for backing at relay chain parent `R1 + 1` and
-subsequently, be included at `R1 + 2` should it be backed successfully. Essentially, a parablock is
-rushing to be backed and included within this two-block window due to the inherent requirement for
-synchrony between the parachain and relay chain.
+The
+[Prospective Parachains subsystem](https://paritytech.github.io/polkadot/book/node/backing/prospective-parachains.html),
```

Comment:
A better way to describe prospective parachains:
Prospective parachains is the relay chain's record of all parablock candidates undergoing the backing and inclusion process. It simultaniously keeps track of candidates from all forks of all parachains.

It is as if you folded the unincluded segments from every fork of every parachain into one giant data structure. When you fold unincluded segments representing different chain forks together, they create a tree structure. Hence the term fragment tree.

A single unincluded segment tells a collator whether it can build on top of one fork of one parachain.

Prospective parachains tells a validator whether it should accept blocks built on top of any fork from any parachain.